.github/workflows/ci.yml

```diff
@@ -42,8 +42,8 @@ jobs:
         if: steps.output-cache.outputs.cache-hit != 'true'
         run: make compile-native
 
-  build:
-    name: Build
+  dialyzer:
```

Comment:
I think it makes sense to keep dializer as part of the "build" since you could think of it as a compile time check. So build = mix compile + dialyzer.

WDYT?

Reply:
We aren't currently running `mix compile` here, and I'm not sure I understand the difference. Isn't it the same as `mix dialyzer --plt`?

Reply:
Ah ok. My point is that I think we should keep the naming of the checks more high level, like "build" or "compile" instead of dialyzer, and "lint " instead of credo. Credo, dialyzer are just specific tools

Reply:
I see. That should simplify later changes to the tools we use.